halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/config/v1/FileWriter.java

```diff
@@ -16,31 +16,21 @@
 
 package com.netflix.spinnaker.halyard.config.config.v1;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
-import org.yaml.snakeyaml.Yaml;
 
 import java.io.*;
-import java.util.Map;
 
 @Component
-public class YamlWriter {
-  @Autowired
-  Yaml yamlParser;
-
-  @Autowired
-  ObjectMapper objectMapper;
-
-  public void write(Object node, String path) throws IOException {
-    write(node, path, false);
+public class FileWriter {
+  public void write(String contents, String path) throws IOException {
+    write(contents, path, false);
   }
 
-  public void write(Object node, String path, boolean append) throws IOException {
+  public void write(String contents, String path, boolean append) throws IOException {
```

Comment:
The problem with being inconsistent is if you never complete (crash, etc) then you have corrupted your file if you were rewriting it or dont know it is corrupted if it is new.